src/styles/designSystem/_typography.scss

```diff
@@ -282,12 +282,12 @@ $titleFontSize: --text-xl;
   @include title-mixin();
 }
 
-$titleSmFontSize: --text-md;
+$titleSmFontSize: --text-xl;
 .title-sm {
   font-family: var(--primary-font-family);
   font-weight: var(--semibold);
   font-size: var($titleSmFontSize);
-  line-height: calc(var($titleSmFontSize) * 24 / 16);
+  line-height: calc(var($titleSmFontSize) * 24.6 / 21.33);
```

Comment:
Are those numbers arbitrary or they were defined on figma?

Reply:
yes they are from figma. I used the ratios so it is more maintainable